crates/swc_ecma_minifier/tests/terser/compress/issue_269/regexp/output.js

```diff
@@ -2,4 +2,4 @@
 /bar/gi;
 RegExp(foo);
 RegExp("bar", ig);
-RegExp("should", "fail");
+/should/afil;
```

Comment:
This is not safe. Polyfill is possible for the RegExp constructor, but it is not possible for RegExp literals.